src/main/java/org/netcracker/educationcenter/elasticsearch/enums/ModelType.java

```diff
@@ -0,0 +1,19 @@
+package org.netcracker.educationcenter.elasticsearch.enums;
+
+/**
+ * Model types. Used in analysis
+ *
+ * @author Mikhail Savin
+ */
+public enum ModelType {
+
+    /**
+     * Jira issue model type
+     */
+    JIRA_ISSUE,
```

Comment:
mb just 'ticket'? because it can be not only from jira

Reply:
done